test/integrations/sources/slack/data.ts

```diff
@@ -0,0 +1,130 @@
+export const data = [
+  {
+    name: 'slack',
+    description: 'Webhook url verificatin event',
+    module: 'source',
+    version: 'v0',
+    input: {
+      request: {
+        body: [
+          {
+            token: 'Jhj5dZrVaK7ZwHHjRyZWjbDl',
+            challenge: '3eZbrw1aB10FEMAGAZd4FyFQ',
+            type: 'url_verification',
+          },
+        ],
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      },
+      pathSuffix: '',
+    },
+    output: {
+      response: {
+        status: 200,
+        body: [
+          {
+            outputToSource: {
+              body: 'eyJjaGFsbGVuZ2UiOiIzZVpicncxYUIxMEZFTUFHQVpkNEZ5RlEifQ==',
+              contentType: 'application/json',
+            },
+            statusCode: 200,
+          },
+        ],
+      },
+    },
+  },
+  {
+    name: 'slack',
+    description: 'Team joined event',
+    module: 'source',
+    version: 'v0',
+    input: {
+      request: {
+        body: [
+          {
+            event: {
+              type: 'team_join',
+              user: {
+                id: 'W012CDE',
+                name: 'johnd',
+                real_name: 'John Doe',
+              },
+            },
+            type: 'event_callback',
+            event_id: 'Ev06TJ0NG5',
+            event_time: 1709441309,
+            token: 'REm276ggfh72Lq',
+            team_id: 'T0GFJL5J7',
+            context_team_id: 'T0GFJL5J7',
+            context_enterprise_id: null,
+            api_app_id: 'B02SJMHRR',
+            authorizations: [
+              {
+                enterprise_id: null,
+                team_id: 'T0GFJL5J7',
```

Comment:
Just wanted to check, if these are actual valid creds, we can use dummy ones.

Reply:
they are dummy credentials